src/components/molecules/NavBar/NavBar.scss

```diff
@@ -246,16 +195,83 @@ $border-radius: 28px;
   &__home-icon {
     cursor: pointer;
     font-size: 24px;
+    color: opaque-white(0.5);
 
     margin-right: $spacing--md;
+  }
 
-    &:hover {
-      font-size: 27px;
-      margin-right: $spacing--sm;
+  &__sparkle {
+    &-logo {
+      margin-right: $spacing--lg;
+      width: $logo-size;
+      height: $logo-size;
+      background-image: url(/assets/icons/sparkle-nav-logo.png);
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: center;
     }
   }
 
   &__profile-popover {
     height: 100%;
   }
+
+  &__radio {
+    &-container {
+      padding: $spacing--lg $spacing--md;
+
+      & > iframe {
+        width: 100%;
+        border: none;
+      }
+    }
+  }
+
+  &__user-avatar {
+    width: $menu-item-size;
+    height: $menu-item-size;
+    margin-right: $spacing--sm;
+
+    img {
+      width: $menu-item-size;
+      height: $menu-item-size;
+    }
+  }
+
+  &__menu {
+    &--volume.mute {
```

Comment:
`&--volume.mute {` -> `&--volume-mute {` or `&--volume-muted {` or `&--muted {`

Benefit: reduce specificity of the style, makes easier to override from parent component(s) if needed.

Reply:
done 0700a1d